tests/integration.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
```

Comment:
@Matt-Esch - I moved the pipe tests into `req.js` and `res.js` since they are against the streams interface. I left the integration tests here, but I removed the tests that were no using this module since they seemed to be testing whether `request` and `nock` are working which is already done in those libs.